package.json

```diff
@@ -16,6 +16,8 @@
     "webpack": "1 || ^2.1.0-beta"
   },
   "devDependencies": {
+    "babel-core": "^6.0.0",
```

Comment:
Oh I was reading you want to get rid of the peer dependency. but yeah if the peer dependency stays then we don't have to add it to dependencies